syft/execution/plan.py

```diff
@@ -607,6 +632,27 @@ def unbufferize(worker: AbstractWorker, protobuf_plan: PlanPB) -> "Plan":
 
         return plan
 
+    @property
+    def code(self) -> str:
```

Comment:
Why is this called code and not __str__?

Reply:
There was something like that in `__str__` before and it was removed. I guess it might be too large and inconvenient for str representation. `.code` is inspired by pytorch's torchscript module :)